pytest/common_lib/contracts.py

```diff
@@ -33,33 +37,24 @@ def build_view_code_request(account_id: str) -> dict:
     }
 
 
-class NetworkRpc(Enum):
-    MAINNET = "https://rpc.mainnet.near.org"
-    TESTNET = "https://rpc.testnet.near.org"
-
+def fetch_testnet_contract() -> bytearray:
```

Comment:
Why not just delete those functions and replace their calls with `current_*_contract()`?